src/views/Home.vue

```diff
@@ -7,15 +7,16 @@
         <div class="home-dialog">
             <div class="card home-dialog__card">
                 <div class="card-content">
-                    <p class="title">“Hello, I'm Mathpar!”</p>
-                    <p class="subtitle">Such beautiful much title</p>
+                    <p class="title">Welcome to Mathpar notebook!</p>
```

Comment:
Can you change "Mathpar notebook" to "Mathpar learning system"? Mathpar notebook is the mathpar.ukma.edu.ua, but our project is a different project located at mathpar.ukma.edu.ua/learning